pkg/ipproto/ipproto_windows.go

```diff
@@ -0,0 +1,10 @@
+package ipproto
+
+import "syscall"
+
+const (
+	TCP    = syscall.IPPROTO_TCP
+	UDP    = syscall.IPPROTO_UDP
+	ICMP   = 0x1
+	ICMPV6 = 0x3a
+)
```

Comment:
The numbers are the same on all platforms (by definition)... if we can't just grab them all from the same syscall-ish package, I'd just write the raw numbers and throw in a comment pointing at https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml